src/rootfs.rs

```diff
@@ -940,4 +938,139 @@ mod tests {
             .get_symlink_args();
         assert_eq!(want, got)
     }
+
+    #[test]
+    #[serial]
```

Comment:
Why does this test need `#[serial]`?

Reply:
hi @utam0k , sorry I do not quite understand about the serial, 
about it, for me about this derive is telling the cargo to do the testing step by step not in parallel.

so please guide if I am wrong 🙇🏻

Reply:
Sure! Your understanding is correct. The `serial` is used for tests that are unstable in parallel depending on the environment in which they are run, such as using the `pipe`. It seems to me that this test does not need to be serial, perhaps with some effort.
Also, here is a script that will run the unit tests over and over again.
https://github.com/containers/youki/blob/main/hack/stress_cargo_test.sh